exporters/variables-scss/src/content/generators/contentGenerator.ts

```diff
@@ -0,0 +1,57 @@
+import { Token, TokenGroup, TokenType } from '@supernovaio/sdk-exporters';
+import { exportConfiguration } from '../../index';
+import { generateCssFromTokens, generateCssObjectFromTokens } from './cssGenerator';
+import { formatCSS } from '../formatters/cssFormatter';
+
+// Add disclaimer to the top of the content
+const addDisclaimer = (content: string): string => {
+  if (exportConfiguration.generateDisclaimer) {
+    return `/* This file was generated by Supernova, don't change manually */\n${content}`;
+  }
+
+  return content;
+};
+
+export const generateFileContent = (
+  tokens: Token[],
+  mappedTokens: Map<string, Token>,
+  tokenGroups: Array<TokenGroup>,
+  tokenTypes: TokenType | TokenType[],
```

Comment:
I think we should do better. You have control over the API of the function and from the transformation I see inside you need to have an array in the input. 

So make the type to `TokenType[]` and leave the transformation somewhere else. The function should do one thing.

The definition of `tokenTypes` is in `fileGenerator.ts` so what about to create the array in `filesData` variable?